docs/contrib/CI.md

```diff
@@ -0,0 +1,81 @@
+# CI Guide
+
+This document describes the pipelines available to the developers and
+code maintainers. Note that pipelines with the suffix `*-Private` are only
+available to code maintainers and The OpenROAD Project members. To access
+Private pipelines one need to be logged in the Jenkins website.
+
+Below there is a list of the available features. Instructions on how to
+navigate Jenkins to access these features are available
+[here](https://docs.google.com/presentation/d/1kWHLjUBFcd0stnDaPNi_pt9WFrrsR7tQ95BGhT1yOvw/edit?usp=sharing).
+
+-   Find your build through Jenkins website or from GitHub.
+-   See test status: Pass/Fail.
+-   Log files for each test.
+-   Build artifacts to reproduce failures.
+-   HTML reports about code coverage and metrics.
+
+## OpenROAD App
+
+-   OpenROAD-Coverage-Public
+    -   Description: run dynamic code coverage tool `lconv`.
+    -   Target: master branch.
+    -   Report link here.
+-   OpenROAD-Coverity-Public
+    -   Description: compile and submit builds to Coverity static code analysis
+        tool.
+    -   Target: master branch.
+    -   Report link here.
+-   OpenROAD-Nightly-Public
+    -   Description: `openroad` unit tests, docker builds, ISPD 2018 and 2019
+        benchmarks for DRT and large unit tests of GPL.
+    -   Target: master branch.
+-   OpenROAD-Public
+    -   Description: `openroad` unit tests and docker builds.
+    -   Target: all branches and open PRs.
+-   OpenROAD-Special-Private
+    -   Description: runs ISPD 2018 and 2019 benchmarks for DRT and large unit
```

Comment:
For developer testing

Reply:
Fixed on [d702c8d](https://github.com/The-OpenROAD-Project/OpenROAD/pull/1740/commits/d702c8dabcbcc16200a3be9db2b96b98ec49c7b9)